components/button-cards/solution-card/solution-card.scss

```
@@ -0,0 +1,141 @@
+.msds-tag {
+  position: absolute;
+  transform: translateY(-50%);
+  border-radius: 20px;
+  right: 20px;
+  display: flex;
+  justify-content: center;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 12px;
+  padding-right: 16px;
+  .msds-icon {
+    width: 16px;
+    height: 16px;
+    margin-right: 8px;
+  }
+}
+
+.msds-solution-card {
+  padding-top: 20px;
+  background-color: #fff;
+  &__header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    &-logo-container {
+      width: 110px;
```

i prefer to do not nest elements definitions inside another element definition and split each definition with empty line so code is easy to read, like that:

```suggestion
.msds-solution-card {
    padding-top: 20px;
    background-color: #fff;
  
    &__header {
      display: flex;
      flex-direction: column;
      align-items: center;
    }
    
    &__header-logo-container {
      width: 110px;
      height: 62px;
      margin-bottom: 28px;
    }
    
    &__header-logo-container-logo {
        height: 100%;
        width: 100%;
    }
}
```

For my is easier to work with this code but do whatever you want man!

---

This comment is applied to the whole scss file!